valve/packet.go

```diff
@@ -14,7 +14,7 @@ import (
 	"time"
 )
 
-const kMaxPacketSize = 1400
```

Comment:
I assume this isn't valid on all engines?

Reply:
Indeed, this would only be the case for [CS:GO SM servers.](https://github.com/alliedmodders/sourcemod/blob/master/extensions/cstrike/rulesfix.cpp#L83) I assume this is still fine since it's a maximum limit?